src/styles/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  box-sizing: border-box;
```

Comment:
It is a good practice to add `box-sizing: border-box;` on all elements `* {...}`